.github/workflows/test.yml

```diff
@@ -34,3 +34,6 @@ jobs:
 
       - name: Run linkinator
         run: npm run test:linkinator
+
+      - name: Run package-lock.json linting
+        run: npm run test:lockfile
```

Comment:
```suggestion
        run: npx lockfile-lint@2.1.2 --type npm --path package-lock.json --allowed-hosts npm github.com
```

Reply:
My rationale for *not* adding github.com to allowed hosts is that packages can be overwritten/republished, which counteracts what we should be aiming for, right?

Reply:
Aside from adding github.com as allowed host, don't you consider it valuable to (1) lock down `lockfile-lint`s dependency tree by installing it as dev dependency and (2) aim for consistency with other similar npm scripts (`test:linkinator | test:html`)?